setup.examples/exim/etc/exim4/conf.d/main/00_mailscanner_listmacrosdefs

```diff
@@ -0,0 +1,72 @@
+# General Exim config for MailScanner Gateway
```

Comment:
Will require testing, I'm not an exim person but at the moment, the default RH install doens't use conf.d, so it will need including in the main config I assume